.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
Alternately, you could just configure projectile to globally ignore `vendor` directories with `projectile-globally-ignored-files` if that suits you better.  

Either way I wouldn't think we'd want to start committing editor-specific files.